# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/cssom-view/scroll-behavior-subframe-window.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/css/cssom-view/scroll-behavior-subframe-window.html.ini
--- a/testing/web-platform/meta/css/cssom-view/scroll-behavior-subframe-window.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/css/cssom-view/scroll-behavior-subframe-window.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -5,8 +5,8 @@
 
   [Main frame with smooth scroll-behavior ; scroll() with auto behavior]
     expected:
-      if debug and (os == "linux") and webrender and not fission: ["PASS", "FAIL"]
-      if debug and (os == "mac"): ["PASS", "FAIL"]
+      if (os == "linux") and debug and webrender and not fission: ["PASS", "FAIL"]
+      if (os == "mac") and debug: ["PASS", "FAIL"]
 
   [Main frame with smooth scroll-behavior ; scrollTo() with auto behavior]
     expected:
@@ -14,11 +14,9 @@
 
   [Main frame with smooth scroll-behavior ; scrollTo() with smooth behavior]
     expected:
-      if (os == "win") and debug and webrender and not swgl: ["PASS", "FAIL"]
-      if (os == "win") and debug and webrender and swgl: ["PASS", "FAIL"]
+      if (os == "win") and debug and webrender: ["PASS", "FAIL"]
 
   [Main frame with auto scroll-behavior ; scrollBy() with default behavior]
     expected:
-      if (os == "win") and debug and webrender and not swgl: ["PASS", "FAIL"]
-      if (os == "win") and debug and webrender and swgl: ["PASS", "FAIL"]
+      if (os == "win") and debug and webrender: ["PASS", "FAIL"]
 